modules/installation-configuration-parameters.adoc

```diff
@@ -779,6 +779,14 @@ On clusters that use Kuryr, {rh-openstack} Octavia does not support availability
 |For compute machines, the availability zone to install root volumes on. If you do not set a value for this parameter, the installer selects the default availability zone.
 |A list of strings, for example `["zone-1", "zone-2"]`.
 
+|`compute.platform.openstack.serverGroupPolicy`
+|Server group policy to apply to the group that will contain the compute machines in the pool. You cannot change server group policies or affiliations after creation. Supported options include `anti-affinity`, `soft-affinity`, and `soft-anti-affinity`. The default value is `soft-anti-affinity`. 
```

Comment:
I think we talked about these on a different PR. The average OpenStack user's going to know exactly what this is, right?

Reply:
Yes. They're not going to be using these parameters without OSP affinity knowledge.